crowbar_framework/lib/crowbar/lock/shared_non_blocking.rb

```diff
@@ -16,6 +16,7 @@
 #
 
 module Crowbar
+  include RemoteNode
```

Comment:
Are you sure you mean `include` and not `require`? These are two really different things, include takes all methods from the `RemoteNode` and includes them into `Crowbar`. The `require` just uses them, which makes more sense to me.